src/app/core/callendar/containers/calendar-container.component.ts

```diff
@@ -38,6 +42,9 @@ export class CalendarContainerComponent implements OnInit {
     }
 
     ngOnInit() {
+        this.store.dispatch(new judgeActions.Get());
+        this.store.dispatch(new fromRoomActions.Get());
```

Comment:
Why are this two here? we should treat this as much not aware of the rest of the system as we can.

Having said that, next few lines shouldn't be here also :(

Reply:
Its a bug fix for https://tools.hmcts.net/jira/browse/SL-2118 
in short when you enter via link to planner, only room view was loaded. Thus judges werent in store so it was displaying null until you go to judges planner and come back